specificatie/openapi.yaml

```diff
@@ -648,6 +810,115 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoek opdracht voor het zoeken naar resource binnen een geometrie.'
+      type: object
+      required:
+        - geometrie
+        - spatialOperator
+      properties:      
+        geometrie:
+          description: 'De geometrie waarbinnen gezocht moet worden naar objecten. Alleen Point en Polygon worden ondersteund. Als Point wordt opgegeven, dan moet ook de paramter radius worden opgegeven zodat in een gebied met een straal rond het opgegeven punt wordt gezocht.' 
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/geometryGeoJSON.yaml'
+        spatialOperator:
+          type: string
+          description: >-
+            "Een uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1. Ondersteund worden:
+            - intersects  : object geometrie `overlapt` of `raakt` de opgegeven contour
+            - within      : object geometrie valt volledig binnen de opgegeven contour
+            - contains    : object geometrie omsluit de opgegeven contour volledig"
+          enum:
+            - intersects
+            - within
+            - contains
+          default: intersects
+        radius:
+          description: 'Als geometrie type point wordt meegegeven, dan kan in een straal rondom dat punt worden gezocht door hier de straal op te geven (eenheid meters).'
```

Comment:
Hier staat "dan kan in een straal rondom dat punt worden gezocht". Maar in de definitie is default 100. Ik lees dat zo dat bij niet opgeven van radius, gezocht wordt met straal 100m. Dus nu is het zo gedefinieerd dat er bij punt altijd gezocht wordt met een straal. 
Moet om zoeken op punt (niet straal) default niet default 0 zijn (en dus minimum ook 0) of bijna 0?

Reply:
Naast deze optie blijft het mogelijk om met een locatie te zoeken, maar dat gebeurt met GET /panden?locatie-query=<point>. Aan die functionaliteit verandert niets.

Reply:
@strijm waar ik op doel is dat de omschrijving suggereert dat zoeken op straal een optie is, terwijl de specificatie zegt dat er altijd met straal wordt gezocht.
Voor panden bestaat zoeken op punt inderdaad ook in de GET als queryparameter, bij adresseerbare objecten niet.
De voorkeur is om ook zoeken op punt mogelijk te maken met de POST operatie. Maar in ieder geval moeten specificatie en omschrijving in lijn met elkaar zijn

Reply:
Ik heb ook een attribuut locatie van type Point toegevoegd, zodat er nu een keuze is tussen, punt, cirkel of polygoon.